docker-entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+nohup java  -cp '/home/sunbird/learner/learning-service-1.0-SNAPSHOT/lib/*' play.core.server.ProdServerStart  /home/sunbird/learner/learning-service-1.0-SNAPSHOT &
```

Comment:
Is this file used? It has reference to learning-service.

Reply:
This file is removed.